src/routes/class-test-route.php

```diff
@@ -0,0 +1,129 @@
+<?php
+/**
```

Comment:
Testing as in, healthcheck test or used in automated testing?

Because any mock or a fixture should be contained in the `tests` folder. This is exposed to public use (without it doing anything really).

Reply:
Routes for automated tests, it's only present in di container used for tests. Not sure how else I'd add a route like this. I can put it in `tests` folder and include it in `class-main.php` for tests.

Reply:
You can leave it in for now like this (if it's not registered in the regular flow it's ok). 

It's just that usually in `src` you'll put anything that you'll use in the production, not tests.